Sources/StreamChatUI/Gallery/ZoomAnimator.swift

```diff
@@ -10,12 +10,6 @@ open class ZoomAnimator: NSObject, UIViewControllerAnimatedTransitioning {
     public weak var fromImageView: UIImageView?
     /// `UIImageView` for view controller being transitioned to.
     public weak var toImageView: UIImageView?
-    /// Snapshot for view controller being transitioned to.
-    public weak var toVCSnapshot: UIView?
-    /// Snapshot for view controller initiating the transition.
-    public weak var fromVCSnapshot: UIView?
-    /// Container view for `transitionImageView`
-    public weak var containerTransitionImageView: UIView?
```

Comment:
This is one of the things I wanted to check with the team. Do you know if is there any customer using any of this ZoomAnimator - Transition API? I didn't find any kind of documentation relating to customization of Zoom transition. Also, while working on this fix, I found these 3 properties being part of what was causing the flicker and weird transition animations issue. I also like the idea to deprecate the properties to avoid breaking changes, if any.

Reply:
To be honest, I think it's highly unlikely this is being used directly, and if it is then I think this change is well worth including. We can separate this update on the CHANGELOG.

This is really nice improvement @hugobernalstream and so nice to see this part of the code improved 💪🏼